src/main/java/com/commercetools/sync/commons/models/NonResolvedReferencesCustomObject.java

```diff
@@ -0,0 +1,56 @@
+package com.commercetools.sync.commons.models;
+
+import io.sphere.sdk.products.ProductDraft;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+public class NonResolvedReferencesCustomObject {
+
+    private String productKey;
+
+    private ProductDraft productDraft;
+
+    private List<String> dependantProductKeys;
```

Comment:
`dependant` is a bit confusing who is dependant on who?

I would name it: `missingReferencedProductKeys`

Reply:
better `set` since it should only contain unique keys?

Reply:
dd6d0e61e82fb516b8e36b4aa9f2106f6dab0b55